jobs/ml-ops/README.md

```diff
@@ -37,14 +37,28 @@ Set your Scaleway access key, secret key and project ID in environment variables
 export TF_VAR_access_key=<your-access-key>
 export TF_VAR_secret_key=<your-secret-key>
 export TF_VAR_project_id=<your-project-id> # you can create a separate project for this example
+```
+
+You can optionally configure non-default CRON schedules to orderly fetch data then train a model, and finally re-load a new model within the inference server. For this, set the following Terraform environment variables:
+
+```console
+export TF_VAR_data_fetch_cron_schedule=<cron-schedule-expression>
+export TF_VAR_training_cron_schedule=<cron-schedule-expression>
+export TF_VAR_inference_cron_schedule=<cron-schedule-expression>
+```
```

Comment:
**nit(unncessary):** we don't need to include the code block for this optional configuration, just the comment above.

Reply:
Removed.